wikimedia/wikipedia-ios
# Fix scrolling bug for large/small text sizes

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T266621

### Notes
* Ultimately a slight rounding issue causes this bug.
* Feel free to push back if you think this is too hacky of a fix.
* See investigation [on Phab task](https://phabricator.wikimedia.org/T266621#6601603).
* While the bug only related to the `isAtTop` calculation, I added padding to the `isAtBottom` calculation under the assumption there could sometimes be rounding issues there, as well.

### Test Steps
1. Use smallest text size.
1. Run app.
1. Open an AaaLD modal. Ensure it is scrolled to the top, as expected.
